src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -269,7 +270,7 @@ public ToolGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 		page1group.addValidationElement(texture);
 
-		name.setValidator(new TextFieldValidator(name, "Tool needs a name"));
+		name.setValidator(new TextFieldValidator(name, L10N.t("elementgui.tool.needs_a_name")));
 		name.enableRealtimeValidation();
 
 		addPage("Visual", pane2);

```

Comment:
Add a translation for the page's names. For the "Triggers" page, use the line I made too.

Reply:
Fixed!